docs/dialect.md

```diff
@@ -173,3 +173,87 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  We can use boolean support functions when a database lacks native boolean support. <br/>
+   **format-true** <br/> 
```

Comment:
@rosswbrown, didn't you change how this works in 2020.3? We should probably have both paths here if so.

Reply:
Yes, there should be a "Tableau 2020.3 and later" which uses the `literal` and `predicate` form.  There should also be the general documentation for 'value' which was the only extension point pre-2020.3.  Details in the CL2186161 description.